lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,63 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * @param {HTMLElement} element
+ * @param {String} attr
+ * @return {String}
+ */
+/* istanbul ignore next */
+function getStyleAttrValue(element, attr) {
+  // Check style before computedStyle as computedStyle is expensive.
+  return element.style[attr] || window.getComputedStyle(element)[attr];
+}
+
+/**
+ * @param {HTMLElement} element
+ * @return {Boolean}
+ */
+/* istanbul ignore next */
+function hasScrollableAncestor(element) {
+  let currentEl = element.parentElement;
+  while (currentEl) {
+    if (currentEl.scrollHeight > currentEl.clientHeight) {
+      const yScroll = getStyleAttrValue(currentEl, 'overflowY');
+      if (yScroll) {
+        return true;
+      }
+    }
+    currentEl = currentEl.parentElement;
+  }
+  return false;
+}
+
+/**
+ * @param {?HTMLElement} element
+ * @return {Boolean}
+ */
+/* istanbul ignore next */
+function isFixed(element) {
```

Comment:
WDYT about `isPositionFixed`?

Reply:
also not great that the caller has to know to bring along all of these other functions, we tried to keep that isolated to just the tap-targets audit which has so much complexity in it already.

others may disagree but since we're not reusing these anywhere else right now, WDYT about defining these helper functions inside `isFixed` so we don't need to think about those pieces? :)